common/app/model/content.scala

```diff
@@ -687,17 +694,11 @@ object Gallery {
     val contentOverrides = content.copy(
       metadata = metadata,
       trail = trail,
-      rawOpenGraphImage = {
-        val bestOpenGraphImage = if (FacebookShareUseTrailPicFirstSwitch.isSwitchedOn) {
-          trail.trailPicture.flatMap(_.largestImageUrl)
-        } else {
-          None
-        }
```

Comment:
I suppose it is there for people to use if they want to.  I would say the conversion uses an implicit but is itself explicit.